tests/test_infer_input_type.py

```
@@ -2,13 +2,52 @@
+
+heavy_modules = {HEAVY_INFERENCE_MODULES!r}
+loaded = sorted(
+    module_name
+    for module_name in sys.modules
+    if module_name in heavy_modules or module_name.startswith("datamodel_code_generator.model.")
+)
+print(json.dumps({{"loaded": loaded, "outcome": outcome}}))
+"""
+    env = os.environ.copy()
+    env.pop("PYTHONWARNINGS", None)
+    return json.loads(subprocess.check_output([sys.executable, "-c", code], env=env, text=True))
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)